Jenkinsfile

```diff
@@ -1,95 +1,97 @@
-
-        if (mainJdk) {
-          // Collect up the jacoco execution results
-          def jacocoExcludes =
-                  // build tools
-                  "**/org/eclipse/jetty/ant/**" + ",**/org/eclipse/jetty/maven/**" +
-                          ",**/org/eclipse/jetty/jspc/**" +
-                          // example code / documentation
-                          ",**/org/eclipse/jetty/embedded/**" + ",**/org/eclipse/jetty/asyncrest/**" +
-                          ",**/org/eclipse/jetty/demo/**" +
-                          // special environments / late integrations
-                          ",**/org/eclipse/jetty/gcloud/**" + ",**/org/eclipse/jetty/infinispan/**" +
-                          ",**/org/eclipse/jetty/osgi/**" + ",**/org/eclipse/jetty/spring/**" +
-                          ",**/org/eclipse/jetty/http/spi/**" +
-                          // test classes
-                          ",**/org/eclipse/jetty/tests/**" + ",**/org/eclipse/jetty/test/**"
-          jacoco inclusionPattern: '**/org/eclipse/jetty/**/*.class',
-                 exclusionPattern: jacocoExcludes,
-                 execPattern: '**/target/jacoco.exec',
-                 classPattern: '**/target/classes',
-                 sourcePattern: '**/src/main/java'
-          consoleParsers = [[parserName: 'Maven'],
-                            [parserName: 'JavaDoc'],
-                            [parserName: 'JavaC']]
-
-          step([$class: 'MavenInvokerRecorder', reportsFilenamePattern: "**/target/invoker-reports/BUILD*.xml",
-                invokerBuildDir: "**/target/its"])
+        stage("Build / Test - JDK11") {
+          agent { node { label 'linux' } }
+          options { timeout(time: 120, unit: 'MINUTES') }
+          steps {
+            mavenBuild("jdk11", "-Pmongodb install")
+            junit '**/target/surefire-reports/TEST-*.xml,**/target/failsafe-reports/TEST-*.xml'
+            warnings consoleParsers: [[parserName: 'Maven'], [parserName: 'Java']]
+          }
         }
 
-        // Report on Maven and Javadoc warnings
-        step([$class        : 'WarningsPublisher',
-              consoleParsers: consoleParsers])
-      }
+        stage("Build Javadoc") {
+          agent { node { label 'linux' } }
+          options { timeout(time: 30, unit: 'MINUTES') }
+          steps {
+            mavenBuild("jdk8", "javadoc:javadoc")
+            warnings consoleParsers: [[parserName: 'Maven'], [parserName: 'JavaDoc'], [parserName: 'Java']]
+          }
+        }
 
-      stage ("Compact3 - ${jdk}") {
-        withMaven(
-            maven: mvnName,
-            jdk: "$jdk",
-            publisherStrategy: 'EXPLICIT',
-            globalMavenSettingsConfig: settingsName,
-            mavenOpts: mavenOpts,
-            mavenLocalRepo: localRepo) {
-          sh "mvn -f aggregates/jetty-all-compact3 -V -B -Pcompact3 clean install -T6"
+        stage("Build Compact3") {
+          agent { node { label 'linux' } }
```

Comment:
here it can be executed on an other node

Reply:
Ditto, same comment as above.